components/src/core/info-list-item/info-list-item.component.scss

```diff
@@ -89,8 +89,8 @@ $denseHeight: 52px;
 
     .pxb-info-list-item-left-content-wrapper {
         order: 2;
-        > * {
-            margin-right: 16px;
+        margin-right: 16px;
```

Comment:
This is the change that is causing the misalignment in the scorecard. We shouldn't set the right margin on the wrapper, we should be setting the right margin on each of the children (otherwise the margin will be there even if there is no content, which is the case in the showcase demo). Does this rule need to be changed? What was the issue you saw that prompted the modification?